static/app/views/dashboardsV2/manage/templateCard.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+import Button from 'sentry/components/button';
+import Card from 'sentry/components/card';
+import {IconAdd, IconGeneric} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import overflowEllipsis from 'sentry/styles/overflowEllipsis';
+import space from 'sentry/styles/space';
+
+type Props = {
+  title: string;
+  widgetCount: number;
+};
+
+function TemplateCard({title, widgetCount}: Props) {
+  return (
+    <StyledCard>
+      <Header>
+        <IconGeneric size="48" />
+        <CardText>
+          <Title>{title}</Title>
+          <Detail>
+            {widgetCount} {t('Starter Widgets')}
```

Comment:
I think you have to include widget count in the string for the localization to make sense
```suggestion
            {t('%s Starter Widgets', widgetCount)}
```

Reply:
🤔 I think that's only the case if we were ever going to do a template with a single widget right?

Reply:
I think we still need to pass the entire context of what's being translated, the position of num widgets might change depending on the language...?

Reply:
oh right, i forgot about that :+1: